c2corg_ui/static/js/documentediting.js

```diff
@@ -488,6 +488,9 @@ app.DocumentEditingController.prototype.hasMissingProps = function(doc, showErro
       hasError = (!doc['lonlat'] || (doc['lonlat'][field] === null || doc['lonlat'][field] === undefined));
     } else if (field === 'date_start') {
       hasError = (doc['date_start'] === null || doc['date_start']  === undefined);
+    // waypoint climbing indoor is the only one that does not require 'elevation'
+    } else if (field === 'elevation' && this.scope['waypoint'] && this.scope['waypoint']['waypoint_type'] === 'climbing_indoor') {
```

Comment:
It would perhaps be more explicit to write
```
} else if (type === 'waypoints' && field === 'elevation' && this.scope['waypoint']['waypoint_type'] === 'climbing_indoor') {
  // waypoint climbing indoor is the only one that does not require 'elevation'
  continue;
}
```

Is ``doc`` the same thing than ``this.scope['waypoint']``? Could we use ``doc['waypoint_type']`` instead of ``this.scope['waypoint']['waypoint_type']``?